### ISSUE TITLE
Not supporting linux/arm64 based runners

### ISSUE THREAD

**Issue by imaddi47-toddleapp** [trstringer/manual-approval#54]
Hi @trstringer,
I am facing this(screenshot) issue on my self-hosted arm64 based linux ec2 instance.
<img width="1056" alt="image" src="https://user-images.githubusercontent.com/114648701/198595838-2cbe7385-0986-4d27-b540-50147afc7d73.png">

I would appreciate if you could deploy docker image with both amd64 and arm64 manifests, currently image with `Dockerfile` configuration on have single arch i.e, amd64.

Hope you will update this soon ✌️. Thanks In advance 🙂

**Comment by trstringer** [comment]
Thanks for opening this issue! Great idea, I'll add ARM support to the backlog. Thanks!

**Comment by Bwvolleyball** [comment]
My branch to add arm support to this project has been open for almost a year 😢

**Comment by OlegSupport** [comment]
We still need arm support for arm runners

**Comment by Bwvolleyball** [comment]
Yep, I agree. My branch remains open and untouched for so long. I think this project might be dead.

**Comment by snskArora** [comment]
Hi @ankit-toddleapp 

This support has been added now.
Closing this issue as it is resolved.
